ouroboros-consensus/src/Ouroboros/Consensus/Storage/VolatileDB/Impl/State.hs

```diff
@@ -414,6 +417,7 @@ mkOpenStateHelper ccfg hasFS checkIntegrity validationPolicy tracer maxBlocksPer
       whenJust mErr' $ \(e, offset) ->
         truncateError file e offset
 
+
```

Comment:
Not needed

Reply:
Not sure about this!

Reply:
As said on Slack, this extra blank line is indeed not needed :slightly_smiling_face: